code/modules/food_and_drink/snacks.dm

```diff
@@ -208,7 +208,36 @@
+	var/static/list/drinkwhitelist = list(/obj/item/reagent_containers/food/drinks/energyshake,/obj/item/reagent_containers/food/drinks/bottle/red,/obj/item/reagent_containers/food/drinks/bottle/blue,/obj/item/reagent_containers/food/drinks/bottle/pink,/obj/item/reagent_containers/food/drinks/bottle/lime,/obj/item/reagent_containers/food/drinks/bottle/spooky,/obj/item/reagent_containers/food/drinks/bottle/spooky2,/obj/item/reagent_containers/food/drinks/bottle/xmas,/obj/item/reagent_containers/food/drinks/bottle/bottledwater,/obj/item/reagent_containers/food/drinks/bottle/softsoft_pizza,/obj/item/reagent_containers/food/drinks/bottle/grones,/obj/item/reagent_containers/food/drinks/bottle/orange,/obj/item/reagent_containers/food/drinks/bottle/gingerale,/obj/item/reagent_containers/food/drinks/bottle/drowsy,/obj/item/reagent_containers/food/drinks/water,/obj/item/reagent_containers/food/drinks/tea,/obj/item/reagent_containers/food/drinks/coffee,/obj/item/reagent_containers/food/drinks/eggnog,/obj/item/reagent_containers/food/drinks/chickensoup,/obj/item/reagent_containers/food/drinks/weightloss_shake,/obj/item/reagent_containers/food/drinks/cola,/obj/item/reagent_containers/food/drinks/cola/random,/obj/item/reagent_containers/food/drinks/peach,/obj/item/reagent_containers/food/drinks/milk,/obj/item/reagent_containers/food/drinks/milk/rancid,/obj/item/reagent_containers/food/drinks/milk/soy,/obj/item/reagent_containers/food/drinks/covfefe,/obj/item/reagent_containers/food/drinks/bottle/beer,/obj/item/reagent_containers/food/drinks/bottle/fancy_beer,/obj/item/reagent_containers/food/drinks/bottle/wine,/obj/item/reagent_containers/food/drinks/bottle/hobo_wine,/obj/item/reagent_containers/food/drinks/bottle/champagne,/obj/item/reagent_containers/food/drinks/bottle/cider,/obj/item/reagent_containers/food/drinks/bottle/rum,/obj/item/reagent_containers/food/drinks/rum_spaced,/obj/item/reagent_containers/food/drinks/bottle/mead,/obj/item/reagent_containers/food/drinks/bottle/vintage,/obj/item/reagent_containers/food/drinks/bottle/vodka,/obj/item/reagent_containers/food/drinks/bottle/tequila,/obj/item/reagent_containers/food/drinks/bottle/gin,/obj/item/reagent_containers/food/drinks/bottle/ntbrew,/obj/item/reagent_containers/food/drinks/bottle/thegoodstuff,/obj/item/reagent_containers/food/drinks/curacao,)
```

Comment:
These are *not* going to get maintained, honestly. 
Not that that's a huge issue because this should probably not be naturally obtainable to begin with.

Reply:
Do vendors ever create their product on destruction? I can't replicate a vending machine exploding its contents.

Would it be acceptable to generate pizzas with a small whitelist of Dan's common snacks etc?

Reply:
I think adding a few more types (read: 1 or 2) of common basic pizza would be well recieved. 
Remember this is a frozen pizza reheating vendor, not a magical school bus of cullinary opportunity. 
the more "wacky" the ingredients, the more i will frown about it.

Reply:
to expand on what i mean by wacky, like - if hacking the vendor gave you access to a chocolate pizza? that might be kinda on-brand. 

if hacking the vendor gives you a Little Danny's Snack Cake pizza, that's kinda out of place. 
Those kinds of abomiations belong to the chef

Reply:
This functionality is now in a `pizza/random/everything` which is probably just a prop